vmware/vmware-cloud-director-ui-components
# Hide actions column in datagrid if there is no contextual actions

## PR Checklist

Please check if your PR fulfills the following requirements:


## PR Type

What kind of change does this PR introduce?




## What does this change do?

Hide the actions column in datagrid if there are no contextual actions available and the contextual actions configuration is set to be displayed in a row.

## What manual testing did you do?
1. Update all contextual actions type from the datagrid example to be `ActionType.STATIC_FEATURED`
2. serve the examples
3. verify that the actions column is missing

## Screenshots (if applicable)
<img width="1484" alt="Screenshot 2023-06-13 at 15 24 04" src="https://github.com/vmware/vmware-cloud-director-ui-components/assets/77797592/1f3b215f-1af9-421a-8c95-db267754b8cf">

## Does this PR introduce a breaking change?




## Other information
